modules/gatk4/variantfiltration/main.nf

```diff
@@ -19,14 +19,14 @@ process GATK4_VARIANTFILTRATION {
     }
 
     input:
-    tuple val(meta), path(vcf)
+    tuple val(meta), path(vcf), path(vcf_index)
     path  fasta
     path  fai
     path  dict
 
     output:
-    tuple val(meta), path("*.vcf"), emit: vcf
-    path "versions.yml"           , emit: versions
+    tuple val(meta), path("*.vcf.gz")	, emit: vcf
+    path "versions.yml"           	, emit: versions
```

Comment:
Please align here by `,`

Reply:
Just to clariify, no output tbi produced?

Reply:
Yes it generates tbi. I added that now